docs/user/openstack/customization.md

```diff
@@ -109,7 +109,17 @@ platform:
       clusterOSImage: http://mirror.example.com/images/rhcos-43.81.201912131630.0-openstack.x86_64.qcow2.gz?sha256=ffebbd68e8a1f2a245ca19522c16c86f67f9ac8e4e0c1f0a812b068b16f7265d
 ```
 
-If the user wants to reuse an existing Glance image without any uploading of binary data, then it is possible to set `clusterOSImage` install config parameter that specifies the Glance image name. In this case no new Glance images will be created, and the image will stay when the cluster is destroyed. In other words, if `clusterOSImage` is not an http(s) URL, then the installer will look into Glance for an image with that name.
+If the user wants to upload the image from the local file system, he can set `clusterOSImage` as `file:///path/to/file`. In this case the installer will take this file and automatically create an image in Glance.
+
+Example:
+
+```yaml
+platform:
+  openstack:
+      clusterOSImage: file:///home/user/rhcos.qcow2
```

Comment:
this is missing documentation in `## Cluster-scoped properties`

Reply:
also the documentation addition is missing from `pkg/types/openstack/platform.go` definition of the field. it needs to provide details that file is also supported.

Reply:
Agree! Done